source/libraries/AccountManager.ts

```diff
@@ -3,7 +3,7 @@ import { Account, generate, privateToAccount } from 'ethjs-account';
 import { Transaction } from 'ethjs-shared';
 import { sign } from 'ethjs-signer';
 import { Connector } from './Connector';
-import { Configuration } from './Configuration';
+import { Configuration } from '../libraries/Configuration';
```

Comment:
This line shouldn't have changed.  Both are correct, but the indirection doesn't add value IMO.